charts/node-analyzer/CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Manual edits are supported only below '## Change Log' and should be used
 exclusively to fix incorrect entries and not to add new ones.
 
 ## Change Log
+
+## 1.8.17
```

Comment:
if you want to manually add changelog entries, please use `# v1.8.17`, doing so will stop the automation from auto generating a new entry, ref: https://github.com/sysdiglabs/charts#pull-requests, https://github.com/sysdiglabs/charts#manual-changelog